### PR TITLE
Landing page like figma

### PR DESCRIPTION
**Pull request by julian1999**

### CODE REVIEW

File: src/components/common/styles.js [hunk 1]
```diff
@@ -16,21 +19,26 @@ export const StyledBoxColumn = styled(StyledBox)`
 export const StyledButton = styled.a`
```
Comment (XianhaiC) [inline src/components/common/styles.js]: Can you avoid modifying StyledButton for now? I still need to fix it. I'll let you know when I do so you can start using it for the landing page.

File: src/components/common/styles.js [hunk 2]
```diff
@@ -30,12 +31,15 @@ export const StyledButton = styled.a`
   ${(props) =>
     props.type === BUTTON_SOLID ?
       props.theme.primaryLight :
-      (props.alt ? props.theme.secondaryAlt : props.theme.secondary)
```
Comment (XianhaiC) [inline src/components/common/styles.js]: Can you keep this line

File: src/components/common/styles.js [hunk 3]
```diff
@@ -30,12 +31,15 @@ export const StyledButton = styled.a`
...
+      props.theme.secondary
 };
   background-color:
-  ${(props) => props.type === BUTTON_SOLID ?
-      (props.alt ? props.theme.secondaryAlt : props.theme.secondary) :
-      'none'
+  ${(props) =>
```
Comment (XianhaiC) [inline src/components/common/styles.js]: This as well

**Review by XianhaiC — CHANGES_REQUESTED** [review]
There seems to still be some changes in styles.js that's overriding what I had written (the alt stuff)

File: src/components/common/styles.js (hunk 1 above)
Comment (julian1999) [reply src/components/common/styles.js]: Sorry about that, I modified it in an earlier commit. I'll find what it was before and leave it alone.

**Review by XianhaiC — APPROVED** [review]
Nice. Go ahead and squash and rebase

**Pull request merged**